core/src/main/java/hudson/model/TaskThread.java

```diff
@@ -188,7 +188,7 @@ public static ListenerAndText forMemory(TaskAction context) {
 
             return new ListenerAndText(
                 new StreamTaskListener(log),
-                new AnnotatedLargeText<TaskAction>(log,Charset.defaultCharset(),false,context)
+                    new AnnotatedLargeText<>(log, Charset.defaultCharset(), false, context)
```

Comment:
Not sure what happened with the formatting here?

Reply:
My guess is the parser got confused (by the `(` & `,` on the previous line?) or has a different opinion about the indentation rules.